DanXiUI/Forum/Views/MentionView.swift

```diff
@@ -80,7 +80,7 @@ struct LocalMentionView: View {
     var body: some View {
         Button {
             model.scrollTo(floorId: floor.id)
-            withAnimation(.smooth) {
+            withAnimation(.spring(bounce: 0.3).speed(1.8)) {
```

Comment:
？

Reply:
This is better but not bouncy, you can try.
The .spring provides a non-linear animation, which is smoother.

Reply:
This bouncing animation differs from native behavior.